service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java

```diff
@@ -71,5 +73,11 @@ public RunImputationGcpJobFlight(FlightMap inputParameters, Object beanBag) {
             flightBeanBag.getRawlsService(),
             flightBeanBag.getImputationConfiguration()),
         dbRetryRule);
+
+    addStep(
+        new FetchOutputsFromDataTableStep(
+            flightBeanBag.getRawlsService(), flightBeanBag.getSamService()));
```

Comment:
oh hmm do we want a different retry rule for external services.  Ive jsut been using thes ame dbRetryRule for all these steps and maybe thats incorrect?